Assets/Vimeo/Scripts/Services/VimeoApi.cs

```diff
@@ -347,29 +363,37 @@ IEnumerator Request(string api_path)
                     if (OnError != null && !IsNetworkError(request)) {
                         OnError(request.downloadHandler.text);
                     }
-                }
-                else if (OnRequestComplete != null) {
+                } else if (OnRequestComplete != null) {
                     OnRequestComplete(request.downloadHandler.text);
                 }
             }
         }
-
+        private void PrepareTusHeaders(UnityWebRequest r, string apiVersion = "3.4")
+        {
+            r.chunkedTransfer = false;
+            r.method = "POST";
+            r.SetRequestHeader("Authorization", "bearer " + token);
+            r.SetRequestHeader("Content-Type", "application/json");
+            r.SetRequestHeader("Accept", "application/vnd.vimeo.*+json;version=" + apiVersion);
+        }
         private void PrepareHeaders(UnityWebRequest r, string apiVersion = "3.4")
         {
             r.chunkedTransfer = false;
-            r.SetRequestHeader("Authorization", "Bearer " + token);
+            r.SetRequestHeader("Authorization", "bearer " + token);
```

Comment:
Was this lower-cased on purpose? I guess it all still works?

Reply:
I guess I followed the Vimeo upload API documentation where they write it with a lower case `b` - https://developer.vimeo.com/api/upload/videos

Reply:
funky - pretty sure it used to be the opposite. Can you lowercase the other `Bearer` lines?

Reply:
Sure